middleware/buffer/source/field.cpp

```diff
@@ -97,7 +97,7 @@ namespace casual
                   }
 
                   if( cursor > end)
-                     common::code::raise::error( common::code::xatmi::argument, "buffer is comprised");
+                     throw std::invalid_argument{ "buffer is comprised"};
                }
```

Comment:
Is "comprised" the correct word? The word "comprise" with past sense "comprised" has the meaning "consists of" eller "be made up of". Is it the word "compromised" that shold be used?  Or perhaps even "corrupt".

Reply:
This error message have been there for many years and in practice it isn't even used for logging or anything so it can even be removed entierly